Source/ZoomNet/Models/CallingPlanType.cs

```diff
@@ -0,0 +1,392 @@
+﻿namespace ZoomNet.Models;
+
+/// <summary>
+/// Represents the type of calling plan.
+/// </summary>
+public enum CallingPlanType
+{
+	/// <summary>
+	/// No feature package.
+	/// </summary>
+	NO_FEATURE_PACKAGE = 1,
```

Comment:
To be consistent with other enums in this library I would prefer if you use Pascal case when naming items. For instance: 
- NoFeaturePackage rather than NO_FEATURE_PACKAGE 
- InternationalTollNumber rather than INTERNATIONAL_TOLL_NUMBER 
- etc.